test/feedback/mailer_test.exs

```diff
@@ -253,13 +253,13 @@ defmodule Feedback.MailerTest do
     end
 
     test "converts 'America/New_York' date to UTC accounting for beginning of daylight saving time" do
-      dt = Util.convert_using_timezone(~N[2020-03-08T02:00:00], "America/New_York")
-      assert Mailer.formatted_utc_timestamp(dt) == "03/08/2020 06:00"
+      dt = Util.convert_using_timezone(~N[2020-03-08T03:00:00], "America/New_York")
```

Comment:
📓 These and all others below are just changes to the ambiguous daylight savings time changes in Timex.